style/pi-hole.css

```diff
@@ -115,3 +115,27 @@ a.lookatme {
     -webkit-animation: fa-spin 1s infinite linear;
     animation: fa-spin 1s infinite linear;
 }
+
+.extratooltipcanvas{
+    -moz-user-select: none;
+    -webkit-user-select: none;
+    -ms-user-select: none;
+}
+#chartjs-tooltip {
+    opacity: 1;
+    position: absolute;
+    background: rgba(0, 0, 0, .7);
+    color: white;
+    border-radius: 3px;
+    -webkit-transition: all .1s ease;
+    transition: all .1s ease;
+    pointer-events: none;
+    -webkit-transform: translate(-50%, 0);
+    transform: translate(-50%, 0);
```

Comment:
These transform lines make the tooltip under the cursor, but cause it to not like being near the right side. If these are removed, then the tooltip does not act so strangely around edges, but it does stick to the right of the cursor:
![screenshot from 2018-04-02 17-07-12](https://user-images.githubusercontent.com/4417660/38216504-f1e95a52-3698-11e8-8a7e-f5b4daeea863.png)
![screenshot from 2018-04-02 17-07-34](https://user-images.githubusercontent.com/4417660/38216506-f546119a-3698-11e8-9cc5-716696c243a2.png)
![screenshot from 2018-04-02 17-07-50](https://user-images.githubusercontent.com/4417660/38216508-f6cdb66c-3698-11e8-8d5a-b9e2057c3bbf.png)
![screenshot from 2018-04-02 17-08-00](https://user-images.githubusercontent.com/4417660/38216509-f837c204-3698-11e8-8872-16ccf1e94fa3.png)